Scripts/datahandling/zonedata.py

```diff
@@ -90,6 +90,8 @@ def __init__(self, data_dir, zone_numbers):
         self.share["share_detached_houses"] = landdata["detach"]
         self["cbd"] = pandas.Series(0, self.zone_numbers)
         self["cbd"].loc[:param.areas["helsinki_cbd"][1]] = 1
+        self["helsinki_other"] = pandas.Series(0, self.zone_numbers)
+        self["helsinki_other"].loc[:param.areas["helsinki_other"][1]] = 1
```

Comment:
I realize now that I have duplicated the "cbd"-dummy row (for "espoo_vant_kau" and "surrounding") without further thinking, and so has @atte-wsp done as well. We now do `.loc[0:upper]` when we should do `.loc[lower:upper]`.

Fortunately the "espoo_vant_kau" has not been used anywhere and "surrounding" dummy affects only the car density model.

Reply:
And I was even thinking that variables of cbd and helsinki_other seemed to be oddly correlated! Never should use copy / paste when coding. Luckily this was easy to fix.